src/smbus/smbus.h

```diff
@@ -33,25 +33,25 @@
 #define SMBUS_ADDR_TYPE_RNG             (3)
 
 #define SMBUS_TRACE_FILTER (0 \
-        | LSHIFT(ERROR) \
-        | LSHIFT(WARN) \
-        | LSHIFT(DEBUG) \
-        | LSHIFT(INFO) \
-        | LSHIFT(INIT) \
+        | BITLSHIFT(1, ERROR) \
+        | BITLSHIFT(1, WARN) \
+        | BITLSHIFT(1, DEBUG) \
+        | BITLSHIFT(1, INFO) \
+        | BITLSHIFT(1, INIT) \
         )
 
 #if 1
 extern const char *smbus_trace_header[];

```

Comment:
Remove unnecessary code/comment.

Reply:
Since different C files use smbus_trace, we cannot remove smbus_trace_header here.

Reply:
Since different C files use smbus_trace, we cannot remove smbus_trace_header here.